src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IHtmlGenerator.cs

```diff
@@ -157,6 +157,7 @@ public interface IHtmlGenerator
             ModelExplorer modelExplorer,
             string expression,
             object value,
+            string format,
```

Comment:
Can we come up with a clever way to not need this? And we can "fix" it all up in 2.0.0?